### PR TITLE
Makefile print section & yaml directory variable

### PR DESCRIPTION
**Pull request by kalbermattenm**
This PR does implement two things:
* Print section for Windows: this is needed because Cygwin handles the existing bash commands in the Windows drive definition, thus it is "not" working. I opted for a specific Windows section, because the behavior differs quite a lot between Linux and Windows systems.
* directory variable: if this variable is defined through bash (in Windows...), then `${directory}` is substituted by `/cygdrive/c/...` everywhere. This does not work when calling the app through Apache (or even pserve) as this one expects to get values like `C:/...` on Windows. Defining `directory` using Python resolves this issue

Please review

### CODE REVIEW

**Comment by kalbermattenm** [comment]
It looks like Travis is trying to use what I wrote... I don't really get why it goes into the ifeq condition (it shouldn't, no ? Isn't it using a Linux architecture...) ?

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -672,14 +672,26 @@ $(VENV_BIN)/flake8: .build/dev-requirements.timestamp
 print: $(PRINT_OUTPUT)/$(PRINT_WAR)
 
 $(PRINT_OUTPUT)/$(PRINT_WAR): $(PRINT_REQUIREMENT)
+ifeq (VENV_BIN, .build/venv/bin)
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: `VENV_BIN` should be `$(VENV_BIN)` :-)

And you should do a rebase to make the tests working :-)

**Comment by kalbermattenm** [comment]
Thanks for the hint @sbrunner ! Travis is now happy...

**Comment by sbrunner** [comment]
Thanks @kalbermattenm :-)

**Pull request merged**